libvast/vast/index_config.hpp

```diff
@@ -0,0 +1,53 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2022 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/type.hpp"
+
+#include <string>
+#include <vector>
+
+namespace vast {
```

Comment:
Should this be in `vast::system` instead seeing that this is probably an implementation detail of the index actor?

Reply:
Per the above thread, it's unrelated to the index actor.